app/models/terms_of_service.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: terms_of_services
+#
+#  id                   :bigint(8)        not null, primary key
+#  changelog            :text             default(""), not null
+#  notification_sent_at :datetime
+#  published_at         :datetime
+#  text                 :text             default(""), not null
+#  created_at           :datetime         not null
+#  updated_at           :datetime         not null
+#
+class TermsOfService < ApplicationRecord
+  scope :published, -> { where.not(published_at: nil).order(published_at: :desc) }
+  scope :live, -> { published.limit(1) }
+  scope :draft, -> { where(published_at: nil).order(id: :desc).limit(1) }
+
+  validates :text, presence: true
+  validates :changelog, presence: true, if: -> { published? }
+
+  def published?
+    published_at.present?
+  end
+
+  def notified?
```

Comment:
Naming thing - could better align this by either calling the query method `notification_sent?` or calling the column in db `notified_at` ?